### ISSUE TITLE
Warning: Could not find any entity matching "pos_player_intermission"

### ISSUE THREAD

**Issue by overvolting** [Unvanquished/Unvanquished#615]
Recently I've seen a lot of these in the ded console.
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"
Warning: Could not find any entity matching "pos_alien_intermission"
Warning: Could not find any entity matching "pos_player_intermission"

**Comment by Viech** [comment]
Are you working on a map or loading one that's unfinished? Every map needs atleast a `pos_player_intermission` entity to function, since that's where the spectator spawns. If there's no `pos_alien_intermission`, which are the alien team's spawn queue spectator positions, it would fallback to `pos_player_intermission`, which could explain why these lines alternate.

**Comment by overvolting** [comment]
I'm not working on a map, however I have been playing on the ones illwieckz re-released. Had no problem playing on it.

**Comment by Kangz** [comment]
You are using custom layouts however, so it might be a bug in the layout code.

**Comment by Viech** [comment]
Maybe the entity names changed and some code references the old/new names while other code looks for both.

**Comment by illwieckz** [comment]
Yes, some old trem maps have this issue (like _Niveus_). Since I'm working on a BSP parser, perhaps it will be possible to rewrite the BSP with an updated _Entities_ part in it, even if the `.map` is missing. It seems to not be an Unvanquished bug, so this issue can probably be closed, no?

**Comment by Viech** [comment]
You can use [this script](http://files.viech.name/scripts/bsp_update_entities.pl) to change entities and their attribute inside a BSP.

**Comment by illwieckz** [comment]
Nice! and my tool is already able to do that now too (example: http://pastebin.com/RP8zhJtD ), but it is able to do more than that :wink: so more maps are coming. :smile:

**Comment by Viech** [comment]
Feel free to also have it replace the deprecated entity names found in [here](https://github.com/Unvanquished/mapeditor-support/blob/master/src/entities/entities.yaml). :grinning:

**Comment by bmorel** [comment]
is that bug still around? I don't think I ever seen it, but maybe it's more related to daemonmap or tooling than to unvanquished?

**Comment by illwieckz** [comment]
This looks to not be a bug but a feature request to  better support incomplete or obsolete maps. Though, in all case I believe it's better to keep the warnings.

**Comment by bmorel** [comment]
I would argue that the fact there's more than one of each warning is a (minor) bug in all cases. That's flooding, and prevents more meaningful messages to be noticed.
Not sure if that aspect is still true, though, thus the "old-needcheck" tag.

I also think silencing warnings is a bad idea, since it only hides problems.

**Comment by bmorel** [comment]
I think the flooding is fixed, since there's a "protection" to skip identical messages when there's too many. And since I think the warning should not be silenced, I'd say we can close this.

**Comment by necessarily-equal** [comment]
By the way, the code handling these "intermission" entities is quite convoluted. I'm quite tempted to rewrite it. I don't think it's worth making small fixes to it without a refactor.
